pkgs/tools/package-management/nix-index/wrapper.nix

```diff
@@ -0,0 +1,19 @@
+{ lib, symlinkJoin, nix-index-unwrapped, makeWrapper, nix }:
+
+if nix == null then nix-index-unwrapped else
+symlinkJoin {
+  inherit (nix-index-unwrapped) name;
+
+  paths = [ nix-index-unwrapped ];
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  postBuild = ''
+    wrapProgram $out/bin/nix-index \
+      --prefix PATH : ${lib.makeBinPath [ nix ]}
+  '';
+
+  meta = nix-index-unwrapped.meta // {
+    hydraPlatforms = [];
```

Comment:
Why? Hydra won't die when it needs to build this tiny package, too.

Reply:
It has `allowSubstitutes = false;` so there's no reason to build it on Hydra, I think? Also, tiny packages may add up.